app/models/event.server.model.js

```diff
@@ -30,7 +30,7 @@ const EventSchema = new Schema({
     free_cancelation: {
```

Comment:
Can we merge the latest master branch into this development branch so the latest changes are reflected? Thanks!